docs/sources/operators-guide/deploy-grafana-mimir/jsonnet/deploying.md

```diff
@@ -100,3 +100,4 @@ You can use [Tanka](https://tanka.dev/) and [jsonnet-bundler](https://github.com
      ```
 
    > **Note**: The generated Kubernetes manifests create resources in the `default` namespace. To use a different namespace, change the `namespace` configuration option in the `environments/default/main.jsonnet` file, and re-generate the Kubernetes manifests.
+   > Jsonnet-based installation uses etcd for the HA tracker by default. To install etcd, we expect you have the [etcd-operator](https://github.com/coreos/etcd-operator) running in your K8S cluster.
```

Comment:
This can be a separate note, since it's unrelated from the previous one.

```suggestion

      > **Note**: Jsonnet-based installation uses etcd for the HA tracker by default. To install etcd, we expect you have the [etcd-operator](https://github.com/coreos/etcd-operator) running in your K8S cluster.
```